app/src/test/java/com/github/h3lp3rs/h3lp/MedicalInformationTest.kt

```diff
@@ -5,39 +5,47 @@ import org.junit.Assert.*
 import java.util.*
 
 class MedicalInformationTest {
+    private val validNumber = "+41216933000"
 
     @Test
     fun badSizeThrowIAE() {
         assertThrows(IllegalArgumentException::class.java) {
-            MedicalInformation(MedicalInformation.MIN_HEIGHT-1,MedicalInformation.MAX_WEIGHT-1,Gender.Male,MedicalInformation.MIN_YEAR+1,"","","",BloodType.ABn)
+            MedicalInformation(MedicalInformation.MIN_HEIGHT-1,MedicalInformation.MAX_WEIGHT-1,Gender.Male,MedicalInformation.MIN_YEAR+1,"","","",BloodType.ABn, "", validNumber)
         }
         assertThrows(IllegalArgumentException::class.java) {
-            MedicalInformation(MedicalInformation.MAX_HEIGHT+1,MedicalInformation.MAX_WEIGHT-1,Gender.Male,MedicalInformation.MIN_YEAR+1,"","","",BloodType.ABn)
+            MedicalInformation(MedicalInformation.MAX_HEIGHT+1,MedicalInformation.MAX_WEIGHT-1,Gender.Male,MedicalInformation.MIN_YEAR+1,"","","",BloodType.ABn, "", validNumber)
         }
     }
     @Test
     fun badWeightThrowIAE() {
         assertThrows(IllegalArgumentException::class.java) {
-            MedicalInformation(MedicalInformation.MAX_HEIGHT-1,MedicalInformation.MIN_WEIGHT-1,Gender.Male,MedicalInformation.MIN_YEAR+1,"","","",BloodType.ABn)
+            MedicalInformation(MedicalInformation.MAX_HEIGHT-1,MedicalInformation.MIN_WEIGHT-1,Gender.Male,MedicalInformation.MIN_YEAR+1,"","","",BloodType.ABn, "", validNumber)
```

Comment:
MedicalInformation should be imported statically. These lines are getting way to long.